src/tests/test_module.py

```diff
@@ -485,29 +485,30 @@ def test_incrby_reset(self):
             r.execute_command('ts.create', 'tester')
 
             i = 0
-            time_bucket = 10
-            start_time = int(time.time())
+            time_bucket = 10*1000
+            start_time = long(time.time()*1000)
             start_time = start_time - start_time % time_bucket
             while i < 1000:
                 i += 1
                 r.execute_command('ts.incrby', 'tester', '1', 'RESET', time_bucket)
 
-            assert r.execute_command('TS.RANGE', 'tester', 0, int(time.time())) == [[start_time, '1000']]
+            assert r.execute_command('TS.RANGE', 'tester', 0, int(time.time()*1000)) == [[start_time, '1000']]
 
     def test_incrby(self):
         with self.redis() as r:
             r.execute_command('ts.create', 'tester')
 
-            start_incr_time = int(time.time())
+            start_incr_time = int(time.time()*1000)
             for i in range(20):
                 r.execute_command('ts.incrby', 'tester', '5')
 
-            time.sleep(1)
-            start_decr_time = int(time.time())
+            start_decr_time = int(time.time()*1000)
             for i in range(20):
                 r.execute_command('ts.decrby', 'tester', '1.5')
 
-            assert r.execute_command('TS.RANGE', 'tester', 0, int(time.time())) == [[start_incr_time, '100'], [start_decr_time, '70']]
+            result = r.execute_command('TS.RANGE', 'tester', 0, int(time.time()*1000))
+            assert result[-1][1] == '70'
+            assert len(result) <= 40
```

Comment:
@gkorland any reference to check the values between `start_incr_time`, `start_decr_time`?

Reply:
done